DogTales/dog/player/PlayerController.hpp

```diff
@@ -0,0 +1,16 @@
+#pragma once
+#include <bave/app.hpp>
+
+namespace dog::player {
+class PlayerController {
+
+  public:
+
+	PlayerController();
+
+	void tick(bave::Seconds dt, const bave::App& app);
+
+	std::unordered_map<std::string, float> key_map{};
```

Comment:
This shouldn't be public, as anybody can modify it. Also, use an annoyingly long form to enable `string_view` lookups:
```cpp
#include <bave/core/string_hash.hpp>

// ...
std::unordered_map<std::string, float, bave::StringHash, std::equal_to<>> key_map{};
```

Reply:
okay, fixed!